cmd/admin-config-setkeys.go

```diff
@@ -0,0 +1,157 @@
+/*
+ * Minio Client (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var adminConfigSetKeysCmd = cli.Command{
```

Comment:
When I run `mc admin config setkeys`, I see usage message corresponding to `mc admin config set`,
```
➜  mc git:(pr2510) mc admin config setkeys
NAME:
  mc admin config set - Set new config file to a Minio server/cluster.

USAGE:
  mc admin config set TARGET

FLAGS:
  --config-folder value, -C value  Path to configuration folder. (default: "/home/kp/.mc")
  --quiet, -q                      Disable progress bar display.
  --no-color                       Disable color theme.
  --json                           Enable JSON formatted output.
  --debug                          Enable debug output.
  --insecure                       Disable SSL certificate verification.
  --help, -h                       Show help.
  
EXAMPLES:
  1. Set server configuration of a Minio server/cluster.
     $ cat myconfig | mc admin config set myminio/


```

Reply:
@krisis,

Unfortunately my changes go/work with @vadmeste's outstanding PR, minio/minio#6113, which is also under review at this point. So, both PRs are supposed to be tested together.

**Note:** I also have more commits on the server side for input validation, which first I've pushed as part of @vadmeste's existing PR, minio/minio#6113, but Anis suggested that it would be cleaner if we separate them. So, there'll be another PR, more server side changes - mostly for input validation, I will be pushing to have the whole feature to work seamlessly for this pod.

Reply:
The usage information is supposed to be displayed when server alias is missing. This is true for both `set` and `setkeys` commands. 
In your example above, `set` command  usage is displayed for `setkeys` command. That's a problem.
I am currently looking into it.

Reply:
Done.